examples/python/controlnet/main.py

```diff
@@ -0,0 +1,141 @@
+#!/usr/bin/env python3
+"""
+Example running ControlNet conditioned on Canny edges.
+
+Based on https://huggingface.co/docs/diffusers/using-diffusers/controlnet.
+
+Run
+```sh
+python 3 examples/python/controlnet/main.py
+```
+"""
+from __future__ import annotations
+
+import argparse
+import os
+
+import cv2
+import numpy as np
+import PIL.Image
+import requests
+import rerun as rr
+import torch
+from diffusers import (
+    AutoencoderKL,
+    ControlNetModel,
+    StableDiffusionXLControlNetPipeline,
+)
+
+
+def controlnet_callback(
+    iteration: int, timestep: float, latents: torch.Tensor, pipeline
+) -> None:
+    rr.set_time_sequence("iteration", iteration)
```

Comment:
```suggestion
    rr.set_time_sequence("iteration", iteration)
    rr.set_time_seconds("timestep", float)
```

Reply:
Added it, it's a bit weird though since this timestep goes the other way. So it'll go from the final image to the noisy one on this timeline.

Reply:
I like it as a way to show that Rerun can handle time in that way